src/widget/notificationscrollarea.cpp

```diff
@@ -66,12 +66,23 @@ void NotificationScrollArea::trackWidget(GenericChatroomWidget* widget)
     }
 }
 
-void NotificationScrollArea::updateTracking()
+/**
+ * @brief Delete notification bar to visible elements on scroll area
+ */
+void NotificationScrollArea::updateVisualTracking() {
+    updateTracking(nullptr);
+}
+
+/**
+ * @brief Delete notification bar to visible elements and to widget on scroll area
```

Comment:
You mean `Delete notification bar from visible elements and widget on sroll area` here?